pkg/kubelet/dockershim/helpers.go

```diff
@@ -375,15 +377,44 @@ func getSecurityOptSeparator(v *semver.Version) rune {
 
 // ensureSandboxImageExists pulls the sandbox image when it's not present.
 func ensureSandboxImageExists(client libdocker.Interface, image string) error {
+	dockerCfgSearchPath := []string{"/.docker", filepath.Join(os.Getenv("HOME"), ".docker")}
```

Comment:
Why is `/.docker` on the search path?

Reply:
It's there to be compatible with `pkg/credentialprovider/config.go:DefaultDockercfgPaths`, the other place that uses docker config to pull images. IIRC, putting docker config in `/` was a common enough (anti?) pattern to warrant support.